commons-numbers-core/src/main/java/org/apache/commons/numbers/core/Precision.java

```diff
@@ -182,7 +182,7 @@ public static boolean equalsIncludingNaN(float x, float y) {
         final boolean yIsNan = Float.isNaN(y);
         // Combine the booleans with bitwise OR
         return (xIsNan | yIsNan) ?
-            !(xIsNan ^ yIsNan) :
+                xIsNan == yIsNan :
```

Comment:
The formatting looks wrong here as you have 8 character indent and then a 4 character indent on the line underneath. It should be:
```java
        return (xIsNan | yIsNan) ?
            xIsNan == yIsNan :
            equals(x, y, maxUlps);
```